example_5_word_cloud/main_1_spacy.py

```diff
@@ -0,0 +1,58 @@
+"""Speakleash: basic wordcloud (using spacy)"""
+
+import os
+from collections import Counter
+
+from speakleash import Speakleash
```

Comment:
Speaklaesh import should be before WordCloud:

```
import matplotlib.pyplot as plt
import spacy
from speakleash import Speakleash
from wordcloud import WordCloud
```

Reply:
Changed to include suggestions